scripts/ci/libraries/_kind.sh

```diff
@@ -392,9 +397,9 @@ function wait_for_airflow_pods_up_and_running() {
         echo "------- Running kubectl get pods: $i -------"
         PODS=$(kubectl get pods --cluster "${KUBECTL_CLUSTER_NAME}" | awk 'NR>1 {print $0}')
         echo "$PODS"
-        NUM_AIRFLOW_READY=$(echo "${PODS}" | grep airflow | awk '{print $2}' | grep -cE '([0-9])\/(\1)' \
+        NUM_AIRFLOW_READY=$(echo "${PODS}" | grep airflow | awk '{print $2}' | grep -cE '([2-9])\/(\1)' \
             | xargs)
-        NUM_POSTGRES_READY=$(echo "${PODS}" | grep postgres | awk '{print $2}' | grep -cE '([0-9])\/(\1)' \
+        NUM_POSTGRES_READY=$(echo "${PODS}" | grep postgres | awk '{print $2}' | grep -cE '([1-9])\/(\1)' \
```

Comment:
Can you explain?

Reply:
Before we would move on before the pods were actually ready. So if the scheduler starts up with 0/2, it would say "ok that's good enough". this makes the function hang until "2/2" are ready for the airflow pod and "1/1" is ready for postgres.

Reply:
I don't think `0/2` would match this:

```bash console
$ echo 'airflow-r7x5g   1/1     Running     0          27d' | grep airflow | awk  '{print $2}' | grep -cE '([0-9])\/(\1)' | xargs
1
$ echo 'airflow-r7x5g   1/2     Running     0          27d' | grep airflow | awk  '{print $2}' | grep -cE '([0-9])\/(\1)' | xargs
0
$ echo 'airflow-r7x5g   0/2     Running     0          27d' | grep airflow | awk  '{print $2}' | grep -cE '([0-9])\/(\1)' | xargs
0
$ echo 'airflow-r7x5g   2/2     Running     0          27d' | grep airflow | awk  '{print $2}' | grep -cE '([0-9])\/(\1)' | xargs
1
```

So it returns `1` iff all pods are in the running state.  (We test the values captured are `== 1` on L404)